learning_wavelets/training_scripts/learnlet_training.py

```diff
@@ -147,21 +187,43 @@ def l_rate_schedule(epoch):
     norm_cback.on_train_batch_end = norm_cback.on_batch_end
 
 
-    n_channels = 1
-    # run distributed
+   # run distributed
     mirrored_strategy = tf.distribute.MirroredStrategy()
     with mirrored_strategy.scope():
-        model = learnlet(input_size=(None, None, n_channels), lr=1e-3, **run_params)
-    print(model.summary(line_length=114))
-
+        model = Learnlet(**run_params)
+        model.compile(
+            optimizer=Adam(lr=1e-3),
+            loss='mse',
+            metrics=[keras_psnr, keras_ssim],
+        )
+        inputs = [tf.zeros((1, 32, 32, 1)), tf.zeros((1, 1))]
+        model(inputs)
+
+    shearlets, _ = cl.get_shearlets(512, 512, 6)
+
+    n_shearlets = np.shape(shearlets)[0]
+    total_size = np.shape(shearlets)[1]
+    half_filters = n_filters//2
+    filter_size = analysis_kernel_size
+    crop_min = total_size//2 - filter_size//2
+    crop_max = total_size//2 + filter_size//2 + 1
+    resized_shearlets = np.zeros((5, filter_size, filter_size, 1, half_filters))
+    for i in range(half_filters):
+        resized_shearlets[:,:,:,0,i] = shearlets[i % n_shearlets, crop_min:crop_max, crop_min:crop_max]
+    
+    filters = np.copy(model.layers[0].get_weights())
+    for i in range(6, 11):
+        filters[i] = resized_shearlets[i-6,:,:,:,:]
+
+    model.layers[0].set_weights(filters)
 
     model.fit(
         im_ds_train,
         steps_per_epoch=200,
         epochs=n_epochs,
         validation_data=im_ds_val,
         validation_steps=1,
-        verbose=0,
+        verbose=1,
```

Comment:
```suggestion
        verbose=0,
```

Let's keep verbose at `0`